### PR TITLE
Fix static map

### PR DESCRIPTION
**Pull request by kparks29**
updated sass as well, wasn't able to install it

### CODE REVIEW

File: src/googleMapService.js [hunk 1]
```diff
@@ -66,6 +67,15 @@ function getStaticMapUrl (addressArray, options) {
 		.concat(options.markerIcon ? 'icon:' + options.markerIcon : '')
 		.concat('|' + encodeURIComponent(addresses))
 		.join('');
+	}
+	
+	var url = getUrl(options, getAddresses(options, addressArray));
+
+	if (url.length >= 2048) {
```
Comment (simpixelated) [inline src/googleMapService.js]: Let's add a comment here about why this check is in place, including this URL:
https://developers.google.com/maps/documentation/staticmaps/#Limits

Also, I think we should use $log.debug inside this conditional, so it's obvious when the map switches to different params that what the user expected.

File: src/googleMapService.spec.js [hunk 2]
```diff
@@ -49,4 +51,38 @@ describe('Cards: GoogleMapService', function() {
...
+				postalCode: zip
+			});
+		}
+		console.log(places.length)
+		expect(GoogleMapService.getStaticMapUrl(places, { addressParams: ['streetAddress', 'city', 'state', 'postalCode'] })).toEqual(jasmine.stringMatching('85203'));
+		expect(GoogleMapService.getStaticMapUrl(places, { addressParams: ['streetAddress', 'city', 'state', 'postalCode'] })).not.toEqual(jasmine.stringMatching('Via'));
+
+	});
+
 });
```
Comment (simpixelated) [inline src/googleMapService.spec.js]: Thanks for adding unit tests!

File: src/googleMapService.js (hunk 1 above)
Comment (kparks29) [inline src/googleMapService.js]: it might have been late, but the $log.debug was failing lint even if i injected it.

**Comment by simpixelated** [comment]
@kparks29 need to fix conflicts before this can be merged

**Comment by simpixelated** [comment]
@kparks29 after talking this over with Sam and Chad, we need to handle this differently when > 2048. Instead of defaulting to uniq zip, we should instead use the first "n" homes until it reaches the limit. We might want to change how this function is organized to have it build the URL in a loop and stop when it reaches the limit.

**Comment by simpixelated** [comment]
@kparks29 @dakiva @cg50x take a look to make sure I didn't break anything on merge, also note the new way to truncate

**Pull request merged**